dist/api_data.py

```diff
@@ -898,7 +898,7 @@ def __ge__(self, other):
         choices=[
         'aggressive_stash_free', 'aggressive_sweep', 'backup_rename', 'checkpoint_evict_page',
         'checkpoint_handle', 'checkpoint_slow', 'checkpoint_stop', 'commit_transaction_slow',
-        'compact_slow', 'conn_close_stress_log_printf', 'evict_reposition', 
+        'compact_slow', 'conn_close_stress_log_printf', 'evict_reposition', 'failpoint_commit',
```

Comment:
I understand this option, but for this issue, we can try the test something as follows:

1. In a release mode, we do not crash if the timestamps are out of order. Let the test run only in the release mode.
2. Ensure that the logging should write to disk when the transaction is committed.
3. Create a table with logging enabled.
4. insert one record with timestamp 10
5. Insert another record with timestamp 5; this commit should fail with the error.
6. Crash and restart the server; this should reply with the log that was written during the commit transaction.
7. Check the data on the table; this should have the old data, as the commit operation failed.

Reply:
Logged table ignores timestamp completely. We need something like mongodb that replays the log to another table. I also don't like the idea that we only run this in the release mode.